### PR TITLE
Add evojax recipe

### PR DESCRIPTION
**Pull request by traversaro**
Add conda-forge recipe for https://github.com/google/evojax .

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by traversaro** [comment]
> I am not even sure if opencv is actually required (see [google/evojax#2](https://github.com/google/evojax/issues/2)),

Upstream confirmed that it is a leftover: https://github.com/google/evojax/issues/2#issuecomment-1038071929, I think we can remove it.

**Comment by traversaro** [comment]
The recipe is ready for review @conda-forge/help-python . It fails on Windows but as it is a `noarch` package that is failing on Windows due to missing dependencies I think the proper thing to do is to just ignore the failure, but correct me this is not the case.

**Comment by BastianZim** [comment]
>  I think the proper thing to do is to just ignore the failure, but correct me this is not the case.

Yes that's correct. :)

File: recipes/evojax/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/evojax-{{ version }}.tar.gz
+  sha256: 739c4dc93f87573b4b0c89219899838a11cc2951365124fdcd74da3c8e8508c3
+  patches:
+    - remove_opencv.patch
```
Comment (BastianZim) [inline recipes/evojax/meta.yaml:12]: FYI: https://github.com/google/evojax/pull/3

**Pull request merged**